KUBERNETES.md

```
@@ -47,4 +47,155 @@ kubectl get hpa -n system-craft
+```bash
+# Delete the plaintext — never commit this
+rm /tmp/raw-secret.yaml
+
+# Commit the encrypted version — safe for Git
+git add kubernetes/sealed-secrets/systemcraft-sealed-secret.yaml
+git commit -m "feat(secrets): add sealed secret for systemcraft"
+```
+
+### How It Works
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)